src/core/client/admin/routes/Moderate/Queue/QueueCommentLeftSubscription.tsx

```diff
@@ -33,9 +33,12 @@ function handleCommentLeftModerationQueue(
   if (connection) {
     const linked = connection.getLinkedRecords("viewNewEdges") || [];
     connection.setLinkedRecords(
-      linked.filter(
-        (r) => r.getLinkedRecord("node")!.getValue("id") !== commentID
-      ),
+      linked.filter((r) => {
+        return (
+          r.getLinkedRecord("node") &&
+          r.getLinkedRecord("node")!.getValue("id") !== commentID
```

Comment:
You can actually write this as:

```suggestion
          r.getLinkedRecord("node")?.getValue("id") !== commentID
```

Though this is related already to the issue that @cvle is already assigned (CORL-1281), so we might need to wait for this fix.